openedx/features/edly/utils.py

```diff
@@ -374,6 +398,20 @@ def get_current_site_invalid_certificate_context(default_html_certificate_config
     return context
 
 
+def get_marketing_url_from_current_site_configurations():
+    """
+    Gets the Marketing URL Root Value from Site Configurations
+
+    Returns:
+        dict: Context data.
+    """
+    context = dict()
+    current_site_configuration = get_current_site_configuration()
+    if current_site_configuration:
+        context['marketing_url'] = current_site_configuration.get_value('MARKETING_SITE_ROOT')
+
+    return context
```

Comment:
we can directly return marketing url here as name suggests without adding it in context dict.